src/app/client_sdk/js_util.ml

```diff
@@ -43,17 +43,34 @@ let payload_common_of_js (payload_common_js : payload_common_js) =
   Signed_command_payload.Common.Poly.
     { fee; fee_token; fee_payer_pk; nonce; valid_until; memo }
 
+let payload_common_to_js
```

Comment:
These changes don't seem to be used. Delete?

Reply:
These functions aren't used here, but should be in this library for the sake of completeness.